core.js

```diff
@@ -0,0 +1,131 @@
+/*$AMPERSAND_VERSION*/
+module.exports = function (xhr) {
+  var result = require('lodash.result');
```

Comment:
Is there a reason for those requires to be inside the exported function?
Unless there are circular dependencies anywhere, there should be no benefit from requiring from within a function that is called later.